### ISSUE TITLE
@ webpack resolve alias – 

### ISSUE THREAD

**Issue by d--j** [vue-styleguidist/vue-styleguidist#149]
Since 1.7.8 ( more accurately since f83faf28d3e07f4b24fd032bd853bbb45e390857 ) vue-styleguidist is defining the `@` alias in such a way that you can only overwrite it with `dangerouslyUpdateWebpackConfig`. Previously that alias was defined in the default config, that gets merged with the project Webpack config. (So the project could overwrite it)

Why does vue-styleguidist defines this alias anyways? I cannot find any usage of it in vue-styleguidist. AFAIK this alias (along with `~`) is quite a common alias for "my project root". Choosing it for vue-styleguidist is maybe not the best idea.

I use this fix for now:
```js
module.export = {
    dangerouslyUpdateWebpackConfig(webpackConfig, env) {
        webpackConfig.resolve.alias['@'] = path.join(__dirname, 'src')
        return webpackConfig
    }
}
```

https://github.com/vue-styleguidist/vue-styleguidist/blob/f83faf28d3e07f4b24fd032bd853bbb45e390857/scripts/make-webpack-config.js#L63-L85

**Comment by rafaesc** [comment]
Hi @d--j 
Yes, you are right, it is not necessary the alias, for that, there is [styleguideComponents option](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Configuration.md#styleguidecomponents)

I've published the fix under v1.7.10.
Please try it and let me know if it works.

**Comment by d--j** [comment]
Hi @rafaesc 
Yes, this fix works.Thanks very much for vue-styleguide and your quick response 👍
